docs/manifest.json

```diff
@@ -161,353 +71,443 @@
-		"parent": "design"
-	},
-	{
-		"title": "Reference",
-		"slug": "reference",
-		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/contributors/reference.md",
-		"parent": "design"
-	},
-	{
-		"title": "Developer Contributions",
-		"slug": "develop",
-		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/contributors/develop.md",
-		"parent": "contributors"
+		"title": "Block Registration ",
+		"slug": "block-registration",
+		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/block-api/block-registration.md",
+		"parent": "block-api"
 	},
 	{
-		"title": "Coding Guidelines",
-		"slug": "coding-guidelines",
-		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/contributors/coding-guidelines.md",
-		"parent": "develop"
+		"title": "Edit and Save",
+		"slug": "block-edit-save",
+		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/block-api/block-edit-save.md",
+		"parent": "block-api"
 	},
 	{
-		"title": "Testing Overview",
-		"slug": "testing-overview",
-		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/contributors/testing-overview.md",
-		"parent": "develop"
+		"title": "Attributes",
+		"slug": "block-attributes",
+		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/block-api/block-attributes.md",
+		"parent": "block-api"
 	},
 	{
-		"title": "Git Workflow",
-		"slug": "git-workflow",
-		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/contributors/git-workflow.md",
-		"parent": "develop"
+		"title": "Deprecated Blocks",
+		"slug": "block-deprecation",
+		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/block-api/block-deprecation.md",
+		"parent": "block-api"
 	},
 	{
-		"title": "Block Grammar",
-		"slug": "grammar",
-		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/contributors/grammar.md",
-		"parent": "develop"
+		"title": "Templates",
+		"slug": "block-templates",
+		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/block-api/block-templates.md",
+		"parent": "block-api"
 	},
 	{
-		"title": "Scripts",
-		"slug": "scripts",
-		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/contributors/scripts.md",
-		"parent": "develop"
+		"title": "Annotations",
+		"slug": "block-annotations",
+		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/block-api/block-annotations.md",
+		"parent": "block-api"
 	},
 	{
-		"title": "Gutenberg Release Process",
-		"slug": "release",
-		"markdown_source": "https://raw.githubusercontent.com/WordPress/gutenberg/master/docs/contributors/release.md",
-		"parent": "develop"
+		"title": "Components",
```

Comment:
I see that components subsections are added at the end, not after this (same for Data module subsections). Is that relevant? My understanding is that it isn't. cc @pento @youknowriad @chrisvanpatten @mkaz

Reply:
The order of sections in the handbook determines the prev/next links. See, for example the nex/prev links of Outreach (the last section of the hanbook as per the sidebar):

![outreach-prev-next-links](https://user-images.githubusercontent.com/583546/55476974-464a7780-5618-11e9-8da9-ca6cad585e50.png)

Note how the next link points to the first package.

Created an issue for this https://github.com/WordPress/gutenberg/issues/14794